frontend/src/views/camp/Collaborators.vue

```diff
@@ -143,8 +146,8 @@ export default {
       return this.collaborators.filter((c) => c.status === 'inactive')
     },
     inviteEmailMessages() {
-      return this.messages.inviteEmail
-        ? Object.values({ ...this.messages.inviteEmail })
+      return this.messages
+        ? Object.values({ ...this.messages.map((value) => serverErrorToString(value)) })
```

Comment:
`serverErrorToString` expects as parameter the error object returned from `hal-json-vuex` and not the violations array. You can directly provide the `e` property from `handleError`. Check for example `DayResponsibles.vue` or `ActivityResponsibles.vue`.

In my opinion we could also move the 409 check into `serverErrorToString`, I guess this would improve error handling for other components, too.

Reply:
serverErrorToString only returns the error.response.data.detail property, which is a concatenated list of the errors.
I would prefer to have the validation errors as a structured object (e.g. a list), that currently vuetify (and maybe later we) can process the individual errors.

Using serverErrorToString on the e variable:
![image](https://user-images.githubusercontent.com/1506818/184597991-31057046-4134-43ed-a0d2-e129da04596d.png)

Using the validation messages in a structured format:
![image](https://user-images.githubusercontent.com/1506818/184600941-2fffda6d-3c06-41ab-80e5-72e4a5e9579d.png)

added :error-count="2" to show 2 errors in https://github.com/ecamp/ecamp3/pull/2897/commits/7bc9cf91a5292afe99bb98c5f66c93c6a8218406

Reply:
As far as i know, we don't have checks for concurrent updates in the backend.
So the 409 error code will currently never occur.

Should i really move it?

Reply:
I'm fine with the list approach, but you're still misusing `serverErrorToString` which expects a complete error object but you throw an item of the violations array at it (which by accident contains a message property) 😀 

Tried & pushed an implementation:
https://github.com/ecamp/ecamp3/pull/2897/commits/b8330c3a984c9d3b278d07988597d8bd3c88fe3d

Reply:
Thank you. :+1: